### DISCUSSION TITLE
LinkedIn OAuth Provider deprecated in favour of LinkedIn (OIDC) Provider

### DISCUSSION THREAD

**Discussion by J0** [supabase/supabase#19204] [Changelog]
LinkedIn has modified the required scopes for their API and OAuth Applications created prior to 1st Aug 2023 do not contain the appropriate scopes. This could cause errors when attempting to sign in with OAuth via LinkedIn. If you have LinkedIn provider enabled on your project a follow up notification will be sent to your email as you could potentially have a LinkedIn OAuth application created before 1st Aug 2023 and be affected. As we don't have access to LinkedIn OAuth configuration we cannot tell with certainty when your OAuth application was created and have to reach out to all users with LinkedIn enabled.

To adjust to this change, we have [introduced a new `LinkedIn (OIDC)` provider](https://github.com/supabase/gotrue/pull/1248) which contains the new required scopes and we have deprecated the existing `LinkedIn` provider.

If you are using a LinkedIn OAuth Application created *before* 1st August 2023 we ask that you [create a new LinkedIn application](https://www.linkedin.com/developers/apps) and migrate your Dashboard credentials from the deprecated `LinkedIn` provider to the new `LinkedIn (OIDC)` provider as shown in the screenshot below. Please do so before 4th Jan 2024 as we will be removing the provider from the dashboard then.

<img width="525" alt="CleanShot 2023-11-25 at 00 51 46@2x" src="https://github.com/supabase/supabase/assets/8011761/b47d6cd3-1593-457a-8b1a-2b2c85c02993">

**Comment by STR1234** [comment]
Does anyone else get an Internal Server Error 500 after migrating to the LinkedIn OIDC flow? Made sure to follow all the steps and just contacted the LinkedIn support. Sadly I do not get any additional info only that an internal server error occurred :'D

**Reply by J0** [reply]
Hey @STR1234,

Sorry to hear about the issues - could we trouble you to [open a ticket](https://supabase.com/dashboard/support/new) if you are still facing the issue?

Let us know

**Comment by J0** [comment]
Hi  Everyone,
As part of the next stage of deprecation, we'll be proceeding to [remove the LinkedIn provider from the dashboard](https://github.com/supabase/supabase/pull/20510)
